### PR TITLE
Use up-to-date Firefox extension and add F-Droid link for Android app.

### PR DESCRIPTION
**Pull request by raf64flo**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| License       | MIT

- Update URL for Firefox extension to Wallabager instead of deprecated extension
- Add F-Droid link for Android app
- Fux a typo

### CODE REVIEW

**Comment by nicosomb** [comment]
Thank you @raf64flo. 

Can you also have a look to docs/fr/user/articles.rst, docs/de/user/articles.rst and docs/it/user/articles.rst please?

**Comment by raf64flo** [comment]
Yep sure.

**Comment by raf64flo** [comment]
@nicosomb Done, even if I'm not sure about the DE and IT translation.

File: docs/de/user/articles.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Du kannst das `Chrome-Addon hier <https://chrome.google.com/webstore/detail/wall
 Android
 """""""
 
-Du kannst die `Android-App hier <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche> herunterladen`_.
+Du kannst die `Android-App hier <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche> herunterladen`_ oder auf `F-Droid <https://f-droid.org/repository/browse/?fdid=fr.gaulupeau.apps.InThePoche>`_.
```
Comment (jlnostr) [inline docs/de/user/articles.rst]: The correct translation would be:

```
Du kannst die `Android-App hier <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche>`_ oder auf `F-Droid <https://f-droid.org/repository/browse/?fdid=fr.gaulupeau.apps.InThePoche>`_ herunterladen.
```

Comment (raf64flo) [inline docs/de/user/articles.rst]: OK going to fix it. Thanks!

File: docs/it/user/articles.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ Usando la vostra applicazione per smartphone
 Android
 """""""
 
-Potete scaricare `qui l'applicazione per Android <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche>`_.
+Potete scaricare `qui l'applicazione per Android <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche>`_ o via `F-Droid <https://f-droid.org/repository/browse/?fdid=fr.gaulupeau.apps.InThePoche>`_.
```
Comment (matteocoder) [inline docs/it/user/articles.rst]: "tramite" is more appropriate than "via" in this context. (sorry for the double comment).

Comment (matteocoder) [inline docs/it/user/articles.rst]: "tramite" is more appropriate than "via" in this context.

**Comment by raf64flo** [comment]
Thanks @matteocoder is it better?

**Comment by matteocoder** [comment]
@raf64flo All good!

**Pull request merged**